### PR TITLE
Started translation of the Italian Language

### PR DESCRIPTION
**Pull request by aspiiire**
I have copied all the english files into the new folder and translated getting_started. 

I have one question though, where I can find code.json inside the french folder in english?

### CODE REVIEW

File: website/i18n/it/docusaurus-plugin-content-docs/current/concepts/combining_providers.mdx [hunk 1]
```diff
@@ -0,0 +1,279 @@
+---
+title: Combining providers
+---
+
```
Comment (rrousselGit) [inline website/i18n/it/docusaurus-plugin-content-docs/current/concepts/combining_providers.mdx:4]: Untranslated files should not be copied. The website will automatically fallback to english

**Review by rrousselGit — CHANGES_REQUESTED** [review]
Thanks! Could you remove english docs clone? :pray:

**Comment by rrousselGit** [comment]
> I have one question though, where I can find code.json inside the french folder in English?

I'm not sure what this is referring to. Could you clarify?

**Comment by aspiiire** [comment]
> > I have one question though, where I can find code.json inside the french folder in English?
> 
> I'm not sure what this is referring to. Could you clarify?

Sure! Under the fr folder I see a file called [code.json](https://github.com/rrousselGit/river_pod/blob/master/website/i18n/fr/code.json). It has some keys like this one "theme.NotFound.title", I have tried to search for it to find the english version by searching that key but I find only the French version.

**Comment by rrousselGit** [comment]
Oh, that's because there is no need for an enligish variant. It is already provided by docusaurus

**Comment by rrousselGit** [comment]
Epic, thanks!

**Pull request merged**